File: Suiki/app/src/main/java/com/suiki/suiki/Model/BddModel/Pathologie.java
package com.suiki.suiki.Model.BddModel;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by Massil on 11/01/2017.
 */

public class Pathologie implements Serializable {
    @SerializedName("nom")
    public String nom;
    @SerializedName("description")
    public String description;
}

File: Service/model/suiki/pathologie.js
var db = require('./../../config/db');

var Schema = db.Schema;

/**
 * Pathologie:
 *  - Nom : Nom de la pathologie
 *  - Description : Description de la pathologie
 */
var pathologie = new Schema({
    nom : 'String',
    description : 'String'
});

module.exports = db.model('Pathologie' , pathologie);